src/griddyn/solvers/sundialsInterface.cpp

```diff
@@ -1,532 +1,534 @@
-/*

```

Comment:
A lot of things happened to this file. I'm tempted to restart from the previous version in cmake_update, just so the git history actually makes sense

Reply:
I'm pretty sure the only 2 real changes are tabs->spaces and static_cast<void*> instead of c-style casts